tool/site/client-react/src/components/conditionalField.tsx

```diff
@@ -1,18 +1,16 @@
-import * as React from "react"
-import {useEffect} from "react"
-import {Field, FieldAttributes, useField} from "formik"
+import React, {useEffect} from "react"
+import {useField} from "formik"
+import SelectField, {SelectFieldProps} from "./SelectField"
 
-interface ConditionalFieldProps extends FieldAttributes<any> {
-  name: string,
+interface ConditionalFieldProps extends SelectFieldProps {
```

Comment:
The changes in this file mean that `ConditionalField` is no longer reusable. Can we do it without changing this file?

Reply:
will look into it